File: math/excel-column-number.cpp
int Solution::titleToNumber(string A) {
    int n=A.size();
    int ans=0;
    int mul=1;
    for(int i=n-1;i>=0;i--){
        ans+= mul*(int(A[i]-'A')+1);
        mul*=26;
    }
    return ans;
}

File: math/excel-column-number.py
class Solution:
    def convert(self, char):
        k = ((ord(char) - ord('A') + 1) % 26)
        if k == 0: k = 26
        return k

    def titleToNumber(self, A):
        ans = 0
        for i in range(len(A)):
            ans += self.convert(A[i]) * (26 ** (len(A)-1 - i))

        return ans

